apps/docs/src/components/PageContent/PageContent.astro

```diff
@@ -19,7 +19,7 @@ const title = frontmatter.title;
+    class="main-section mb-[4rem] [&>_*+*]:mt-2.5 [&_p]:text-sm [&_p]:leading-relaxed [&_p]:text-theme-gray-400 [&_p]:dark:text-theme-gray-800 md:[&_p]:text-base [&_h1]:mb-2 [&_h2:not(nav_h2)]:mb-2 [&_h2]:font-bold [&_h2]:leading-none [&_h2:not(nav_h2)]:text-xl [&_h3]:mb-2 [&_h3]:font-bold [&_h3]:leading-none [&_h3]:text-md md:[&_h3]:text-lg [&_h4]:mb-2 [&_h4]:font-bold [&_h4]:leading-none [&_h4]:text-md [&_h5]:mb-2 [&_h5]:font-bold [&_h5]:leading-none [&_h5]:text-sm [&_h6]:mb-2 [&_h6]:font-bold [&_h6]:leading-none [&_h6]:text-xs [&_h4:not(:first-child)]:mt-[2rem] [&_h5:not(:first-child)]:mt-[2rem] [&_h6:not(:first-child)]:mt-[2rem] [&_h2:not(:first-child)]:mt-[3rem] [&_h3:not(:first-child)]:mt-[3rem] [&_ul]:leading-4 [&_ul]:text-theme-gray-400 [&_ul]:dark:text-theme-gray-800 [&_a:not(nav_a)]:text-theme-accent [&_li:not(nav_li):not(:first-child)]:mt-[0.75rem] [&_ul:not(nav_ul)]:pl-2 [&_ol:not(nav_ol)]:pl-2 [&_iframe]:w-full [&_iframe]:h-auto [&_iframe]:aspect-video [&_a:hover]:underline [&_a:focus-visible]:underline [&_a:focus-visible]:outline [&_a:focus-visible]:outline-2 [&_a:focus-visible]:outline-offset-4 [&_a:focus-visible]:outline-theme-accent [&_code]:text-xs [&_code:not(pre_code)]:bg-theme-gray-950 dark:[&_code:not(pre_code)]:bg-theme-gray-100 [&_code]:py-[0.2rem] [&_code]:px-[0.33rem] [&_code]:my-[-0.2rem] [&_code]:mx-[-0.125em] [&_code]:rounded-[3px] [&_code]:break-words [&_pre]:relative [&_pre]:py-2 [&_pre]:pl-4 [&_pre]:pr-[4rem] [&_pre]:text-xs md:[&_pre]:text-sm [&_pre]:overflow-x-auto [&_pre]:overflow-y-hidden [&_pre]:-mx-4 md:[&_pre]:mx-0 md:[&_pre]:rounded-lg md:[&_pre]:pl-2.5 md:[&_pre]:pr-5 [&_table]:relative [&_table]:py-2 [&_table]:pl-4 [&_table]:pr-[4rem] [&_table]:text-xs [&_table]:overflow-x-auto [&_table]:overflow-y-hidden [&_table]:m-0 md:[&_table]:rounded-lg md:[&_table]:pl-2.5 md:[&_table]:pr-5 [&_table]:w-full [&_table]:px-0 [&_table]:border-collapse [&_tr:nth-of-type(odd)]:bg-theme-gray-950 dark:[&_tr:nth-of-type(odd)]:bg-theme-gray-400 [&_th]:font-bold [&_th]:p-1 [&_td]:p-1"
```

Comment:
removed ```[&_h1]:font-black [&_h1]:leading-none [&_h1]:text-xxl``` - it overwrote the styles of every h1 on page along with h1 in hero component. I'm not 100% sure I could do it, but it seems to work the same as before

Reply:
yes you could, docs are not designed yet so we can mess around with its styles if needed. Ideally they won't interfere with component styles at all so do anything that's necessary to help the components dev! :D